kobodl/kobo.py

```diff
@@ -166,7 +184,7 @@ def __GetExtraLoginParameters(self) -> Tuple[str, str, str]:
             )
         requestVerificationToken = html.unescape(match.group(1))
 
-        return koboSignInUrl, workflowId, requestVerificationToken
+        return koboSignInUrl, workflowId, requestVerificationToken, authCookie
```

Comment:
Don't forget to update the return type for `__GetExtraLoginParameters`  to `Tuple[str, str, str, str]`